app/services/liquid_assigns_service.rb

```diff
@@ -6,7 +6,7 @@ def initialize(project)
   end
 
   def assigns
-    result = { 'project' => ProjectDrop.new(project) }
+    result = project_drops
```

Comment:
what about `project_assigns` instead to be consistent with the rest of the methods?